src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -258,7 +258,7 @@ extension DynamicTableViewController {
 
 	func tableView(_: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
 		if dynamicTableViewModel.section(at: indexPath).isHidden(for: self) {
-			return 0
+			return .leastNonzeroMagnitude
```

Comment:
Why that change?

Reply:
Returning 0 at this point may result in the default height rather than 0.